pages/index.html

```diff
@@ -0,0 +1,93 @@
+                    <li class="padding"><a class="pad" href="./index.html">Home</a></li>
+                    <li class="padding"><a class="pad" href="./about.html">About</a></li>
+                    <li class="padding"><a class="pad" href="#">Contact</a></li>
+                    <li class="padding"><a class="pad" href="#">Join</a></li>
+                  </ul>
+
+                <!-- desktop menu -->
+                <div class="desktop-menu">
+                    <img id="logo" src="../img/logo.jpg" alt="logo of company">
+                    <a href="../pages/index.html">Home</a>
+                    <a href="./about.html">About</a>
+                    <a href="#">Program</a>
+                    <a href="#">Join</a>
+                    <a href="#">Sponsor</a>
+                    <a href="#">Campaign</a>
+
+                </div>
+                
+            </nav>
+        </div>
+        <header class="flex-column  bg">
+        
+            <div class="margin-header">
+                <h3 class="orange-color" id="header-text-1">
+                    "Hello people from all round the world"
+                </h3>
+                <h1 class="orange-color header-text2">GET INSPIRED BY THE BEST PEOPLE FROM AROUND THE WORLD ABOUT CLIMATE CHANGE</h1>
+            </div>
+            <p class="paragraph">This is Commitment to the people who have worked day and night to pass.</p>
+            <div class="margin-header ">
+                <h3 class="hashtags">#saveEarth #GreenCare #development</h3>
+                <h4 class="last-header-text ">@ For a better tomorrow that we will work for our kids </h4>
+            </div>
+            </header>
+
+        <section class="section-topic">
+            <h1 id="tile-section-2">Topic</h1>
+  <div id="desktop-variation-1">
+      <article class="article margin-header"><img class="article-img" src="../img/emb.png" alt="icon"><h3 class="text-overflow article-text ">Embedded systems</h3>
+      </article>
+      <article class="article margin-header"><img class="article-img" src="../img/Protocol.png" alt="icon"><h3 class="text-overflow article-text">Protocols</h3>
+      </article>
+      <article class="article margin-header"><img class="article-img" src="../img/OS.png" alt="icon"><h3 class="text-overflow article-text">Operating Systems</h3>
+      </article>
+      <article class="article margin-header"><img class="article-img" src="../img/progra.png" alt="icon"><h3 class="text-overflow article-text">Programming languages</h3>
+      </article>
+      <article class="article margin-header"><img class="article-img" src="../img/algo.png" alt="icon"><h3 class="text-overflow article-text">Algorithms</h3>
+      </article>
+
+  </div>
+        
+            
+
+        </section>
+        <section class="speakers-section"> 
+            <h1 id="speaker-head">Speakers</h1>
+            <div class="bar"></div>
+            <div class="desktop-v"></div>
```

Comment:
- [ ] Please put the articles in the middle as they are more to the left side to make your design more organized.
![Screenshot from 2022-04-22 19-42-26](https://user-images.githubusercontent.com/33752735/164766738-e660eea8-94ac-4fb9-8dd9-dbeaa2ff658c.png)

- [ ] Kindle be noticed that every article has two columns, the first with the image at the left and the second with the title, subtitle, and description.

- [ ] Please make your design on mobile more responsive.
![Screenshot from 2022-04-22 19-45-51](https://user-images.githubusercontent.com/33752735/164767869-51c94e79-9435-4f19-a26c-445d35755d07.png)